elinks (0.9.1-1.kon.1) local; urgency=low

  * Local version!
  * src/protocol/http/http.c (http_send_header):
    Corrected commas in Accept-Charset.

 -- Kalle Olavi Niemitalo <kon@iki.fi>  Wed,  7 Apr 2004 10:00:12 +0300

--- elinks_0.9.1-1/src/protocol/http/http.c	2004-01-01 17:36:18.000000000 +0200
+++ elinks_0.9.1-1_aclen/src/protocol/http/http.c	2004-04-07 09:53:19.000000000 +0300
@@ -508,11 +508,10 @@ http_send_header(struct connection *conn
 
 		if (init_string(&ac)) {
 			unsigned char *cs;
-			int aclen = 0;
 			int i;
 
 			for (i = 0; (cs = get_cp_mime_name(i)); i++) {
-				if (aclen) {
+				if (ac.length) {
 					add_to_string(&ac, ", ");
 				} else {
 					add_to_string(&ac, "Accept-Charset: ");
